docs/security.md

```diff
@@ -3,96 +3,12 @@ post_title: Security
 menu_order: 40
 enterprise: 'no'
 ---
-# Mesos Security
 
-## SSL
-
-<table class="table">
-  <tr>
-    <td>
-      `security.mesos.ssl.enabled`
-    </td>
-
-    <td>
-      Set to true to enable SSL on Mesos communication (default: false).
-    </td>
-  </tr>
-</table>
-
-
-## Authentication
+This topic describes how to configure security for Spark.
```

Comment:
@mgummelt this particular topic merely points to the Spark service authentication information, and then discusses SSL. how about `authentication` instead of `security`?